src/main/java/uk/gov/hmcts/darts/event/controller/EventsController.java

```diff
@@ -90,7 +94,17 @@ public ResponseEntity<List<CourtLog>> courtlogsGet(
         List<CourtLog> courtLogs = courtLogsService.getCourtLogs(courthouse, caseNumber, startDateTime, endDateTime);
 
         return new ResponseEntity<>(courtLogs, HttpStatus.OK);
+    }
+
+    @Override
+    @SecurityRequirement(name = SECURITY_SCHEMES_BEARER_AUTH)
+    @Authorisation(contextId = ANY_ENTITY_ID,
+        globalAccessSecurityRoles = {SUPER_ADMIN})
+    public ResponseEntity<EventMapping> eventMappingsEventHandlerIdGet(Integer eventHandlerId) {
```

Comment:
You can give the controller method names a nicer name by specifying the `operationId` in the open api spec